### PR TITLE
BLD-9056- activating end chat menu

### PR DESCRIPTION
**Pull request by tehilaR2**
activity controls chat logic

### CODE REVIEW

File: CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+            }
+
+        } catch (ex: Exception) {
+            ex.printStackTrace()
+        }
+
+    }
+
+    override fun onDestroy() {
+        clearAllResources()
```
Comment (ozbenoved) [inline CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt:137]: Why clearing the resources also here?

File: CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+    }
+
+    //-> previous listener method signature @Override onPhoneNumberNavigation(@NonNull String phoneNumber) {
+    override fun onPhoneNumberSelected(phoneNumber: String) {
+        try {
+            val intent = Intent(Intent.ACTION_DIAL)
+            intent.data = Uri.parse("tel:$phoneNumber")
+            startActivity(intent)
+        } catch (e: ActivityNotFoundException) {
+
```
Comment (ozbenoved) [inline CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt]: "Log.w(CONVERSATION_FRAGMENT_TAG, "failed to open dialer" + e.message)" ?

**Review by ozbenoved — APPROVED** [review]

File: CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt (hunk 1 above)
Comment (tehilaR2) [reply CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt:137]: onDestroy is a sure event

File: CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt (hunk 2 above)
Comment (tehilaR2) [reply CombinedExDemo/App/demo/src/main/java/com/conversation/demo/ChatActivity.kt]: done

**Pull request merged**